### DISCUSSION TITLE
Retrocession: Emergency Stop

### DISCUSSION THREAD

**Discussion by HakunMatat4** [KlipperScreen/KlipperScreen#1304] [General]
I just updated KlipperScreen to its latest version and for some odd reason, Emergency Stop was moved to within power section...??????
Imo that is a big retrocession, there's a reason of why it's called emergency stop.

Imagine if a CNC machine button was kept covered by a cover with key.
Or something went sideways, you go to hit emergency button on your printer only to find out you need a second screen to access/stop it.

Anyway, I think this is bad and have to be moved back to its own shortcut on the main screen for obvious reasons.

**Comment by alfrix** [comment]
Currently it will show during printing which would be the normal use case, otherwise within the sub-menu.
a lot of people wanted a confirmation dialog, that is also not ideal, but consider that most 3d printers are not really dangerous as lathe or something like that, in general is only used to cancel a heat-up event
if the machine is dangerous you should consider having a physical e-stop button.

**Reply by HakunMatat4** [reply]
There's a reason why it's called emergency stop, this is the first time an emergency stop was removed because of not being important during non printing operations. 

When calibrating a probe, there was a need to emergency stop the printer so now you need to go to a menu first before reaching emergency stop while the toolhead is digging your sh*t

This approach is wrong in so many levels.
Anyway.

**Reply by jordanruthe** [reply]
I agree. Emergency stop is a safety issue and even though printers are safe, anything can happen. This should be an opt out feature, not something where the stop is hid by default. Personally, I'd Rather have people ruin prints by accident. In a real emergency, sub seconds could make a difference and any movement, even without printing, could have a major affect.
